app/channels/application_cable/channel.rb

```diff
@@ -1,7 +0,0 @@
-# Be sure to restart your server when you modify this file. Action Cable runs in a loop that does not support auto
-# reloading.
-
-module ApplicationCable
-  class Channel < ActionCable::Channel::Base
```

Comment:
🗒️ These ActionCable files rely on the `action_cable` gem being loaded. We explicitly do _not_ load ActionCable in our `application.rb`. That means that these files will cause a crash if/when this app is deployed to production.

We can't have it both ways: we either need to load the `action_cable` gem, or we need to remove these non-functional classes if we want to have an app that works out of the box.

I chose to remove the classes.